public/app/features/alerting/unified/utils/labels.ts

```diff
@@ -0,0 +1,5 @@
+import { Labels } from '../../../../types/unified-alerting-dto';
+
+export function labelsToTags(labels: Labels) {
+  return Object.entries(labels).map(([label, value]) => `${label}=${value}`);
```

Comment:
Can we sort these in lexicographical order? I've noticed the UI makes the labels jump around after consecutively evaluating (probably because the order in which we receive them isn't deterministic)

Reply:
Yes, good point!